diff --git a/node_modules/parse-server/lib/RestQuery.js b/node_modules/parse-server/lib/RestQuery.js
index 1155c10..ab368d0 100644
--- a/node_modules/parse-server/lib/RestQuery.js
+++ b/node_modules/parse-server/lib/RestQuery.js
@@ -242,8 +242,8 @@ _UnsafeRestQuery.prototype.execute = function (executeOptions) {
   return Promise.resolve().then(() => {
     return this.buildRestWhere();
   }).then(() => {
-    return this.denyProtectedFields();
-  }).then(() => {
+//     return this.denyProtectedFields();
+//   }).then(() => {
     return this.handleIncludeAll();
   }).then(() => {
     return this.handleExcludeKeys();
